File: task-03/Python/SubTask4.py
in_file = open('input.txt', 'r')
n = int(in_file.read().strip())

out_file = open('output.txt', 'w')

for i in range(n):
    out_file.write(' ' * (n - i - 1) + '*' * (2 * i + 1) + '\n')

for i in range(n - 2, -1, -1):
    out_file.write(' ' * (n - i - 1) + '*' * (2 * i + 1) + '\n')

File: task-03/Ruby/SubTask4.rb
n = File.read('input.txt').to_i

File.open('output.txt', 'w') do |outfile|
  n.times do |i|
    outfile.puts ' ' * (n - i - 1) + '*' * (2 * i + 1)
  end
  (n-1).times do |i|
    outfile.puts ' ' * (i + 1) + '*' * (2 * (n - i - 2) + 1)
  end
end
